doc/schema/attribute-types/datetime.md

```diff
@@ -15,92 +15,84 @@ Use [dateonly] for a date type without time.
-  sets the step size to allow dates every other hour: `datetime({step: 1000*60*60*2})`
+Dateonly and datetime data will be returned as `1990-12-31T06:00:00.000Z`, `1990-12-31`, or `null` as follows:
 
-## Form Controls
-
-`datetime()` will result in a [`<input type="datetime-local">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) control
-with minute resolution.
-
-`datetime({step: "day"})` in a [`<input type="date">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) control with day resolution.
-
-`datetime({step: "week"})` in a [`<input type="week">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/week) control with week resolution.
-
-`datetime({step: "month"})` in a [`<input type="month">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month) control with month resolution.
-
-`datetime({step: "month"})` in a [`<input type="month">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month) control with month resolution.
+```js
+{
+  data: {
+    ...
+    attributes: {
+      ...
+      dueDate: "1990-12-31T06:00:00.000Z"
+    }
+  }
+}
+```
 
-Currently, any other step value will result in a [`<input type="datetime-local">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) control which will be validated after the value is set.
+### Mutating Data
 
+When creating or updating a datetime or dateonly attribute, a valid date in the form of `1990-12-31T06:00:00.000Z`, `1990-12-31`, or `null` must be provided. Any other value type will return a service error.
 
+When `datetime` is given a step of `"day"`, a valid date must be in the form of `1990-12-31`, or `null`, Any other value type will return a service error.
 
-<details>
-<summary>
+Note: Both formats work with both date types. If a `datetime` attribute is created/updated with `1990-12-31` the returned value will be `1990-12-31T06:00:00.000Z`. For `dateonly`, the opposite is true. Any time portion used in creating/updating the attribute will be truncated.
 
-## Advanced Details
+## React Rest Behavior
 
-</summary>
+Similar to the middleware, you MUST provide react rest models a valid date in the form of `1990-12-31T06:00:00.000Z`, `1990-12-31`, or `null` value. Likewise, they will always return these values:
 
-### Control Type
+```ts
+Todo.createOne({ attributes: { dueDate: "1990-12-31T06:00:00.000Z" } })
 
-```js
-{
-  type: "Date",
-  allowNull: Boolean, 
-  min: ISODateString | -Infinity,
-  max: ISODateString | Infinity,
-  step: DateStepNumber | DateStepString
-}
+const [todo, todoMeta] = hatchedReactRest.Todo.useOne({ id })
+todo.dueDate //-> "1990-12-31T06:00:00.000Z", "1990-12-31", null, or undefined
 ```
 
+## Grid Behavior
 
-### Sequelize Type
+The text date values will be presented in the grid. If the value is `null` or `undefined`, no value will be presented in the grid.
```

Comment:
should mention the datetime will be formatted to your locale, outside of usa it will _usually_ be `dd mm yyyy` and screenshots are `mm dd yyyy`

Reply:
Great call out